pkgs/development/python-modules/libnacl/default.nix

```
@@ -17,6 +18,15 @@ buildPythonPackage rec {
     sha256 = "sha256-nttR9PQimhqd2pByJ5IJzJ4RmSI4y7lcX7a7jcK+vqc=";
   };
 
+  patches = [
+    # Fixes build on 32-bit platforms
+    (fetchpatch {
+      name = "fix-crypto_kdf_derive_from_key-32bit.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/saltstack/libnacl/pull/130.diff";
```

please use the commit patch
```suggestion
      url = "https://github.com/saltstack/libnacl/commit/e8a1f95ee1d4d0806fb6aee793dcf308b05d485d.patch";
```

---

@marsam Thanks, updated the PR.